docs/cloud/details/access_management/abac.md

```diff
@@ -13,7 +13,7 @@ permalink: /docs/cloud/abac
 {: .note }
 This feature may not be available in your environment. Please contact sales.
 
-Grants, purposes, column masks and row-level filters can all be made dynamic. This means that you can define boolean expressions to dynamically calculate the data objects (for grants and masks) and beneficiaries (for all) that should be included in the access control.
+Grants, column masks and row-level filters can all be made dynamic. This means that you can define boolean expressions to dynamically calculate the data objects (for grants and masks) and beneficiaries (for all) that should be included in the access control.
```

Comment:
I don't think row filters can be dynamic since they can only apply to a single data object

Reply:
Ah, just saw the next change…